cmd/ndnping-dpdk/README.md

```diff
@@ -1,20 +1,22 @@
 # ndnping-dpdk
 
-This program acts as [ndnping](https://github.com/named-data/ndn-tools/tree/master/tools/ping) client or server on specified interfaces.
-It can serve a traffic generator to benchmark a forwarder or a network.
+This program acts as an [ndnping](https://github.com/named-data/ndn-tools/tree/master/tools/ping) client or server on the specified interfaces.
+It can serve as a traffic generator to benchmark a forwarder or a network.
 
 ## Usage
 
-```
+```sh
 sudo ndnping-dpdk EAL-ARGS -- [-initcfg=INITCFG] [-tasks=TASKS] [-cnt DURATION]
 ```
 
 **-initcfg** accepts an initialization configuration object in YAML format.
-This program recognizes *mempool* section only.
+This program recognizes the `Mempool` section only.
+See [here](../../docs/init-config.sample.yaml) for an example.
 
 **-tasks** accepts a task description object in YAML format.
+See [here](../../docs/ndnping.sample.yaml) and below for examples.
 
-**-cnt** specifies duration between printing counters.
+**-cnt** specifies the time interval (FIXME: what unit?) between printing counters (FIXME: default value?).
```

Comment:
how should I fill these two FIXMEs?

Reply:
The unit is part of the flag. It has to be a string acceptable to `time.ParseDuration` function, which must contain the unit.
See https://golang.org/pkg/flag/#Duration and https://golang.org/pkg/time/#ParseDuration

---

Do not specify default value in README. User can view default value using `-h` flag.
`-h` flag is implicitly declared by Package flag.